routes/web.php

```diff
@@ -3,8 +3,10 @@
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\Auth\LoginController;
 use App\Http\Controllers\Admin\UserController;
+use App\Http\Controllers\Manager\MemberController;
 use App\Http\Controllers\Manager\TaskController;
 use App\Http\Controllers\Manager\TeamController;
+use App\Http\Controllers\TeamController as ControllersTeamController;
```

Comment:
```suggestion
use App\Http\Controllers\Manager\TeamController  as ManagerTeamController;
use App\Http\Controllers\TeamController;
```

もしも別名をつけるならこういう感じの名前付けの方がわかりやすいかもしれませんね。

Reply:
学習メモのためにも、後者でやってみます。